document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresDatastore.java

```diff
@@ -72,7 +73,12 @@ public Set<String> listCollections() {
       DatabaseMetaData metaData = client.getConnection().getMetaData();
       ResultSet tables = metaData.getTables(null, null, "%", new String[] {"TABLE"});
       while (tables.next()) {
-        collections.add(database + "." + tables.getString("TABLE_NAME"));
+        Optional<String> nonPublicSchema =
+            Optional.ofNullable(tables.getString("TABLE_SCHEM"))
```

Comment:
Typo?
TABLE_SCHEMA

Reply:
It's not, but thanks for pointing it out - certain deserves a comment.

> TABLE_CAT String => table catalog (may be null)
TABLE_SCHEM String => table schema (may be null)
TABLE_NAME String => table name
TABLE_TYPE String => table type. Typical types are "TABLE", "VIEW", "SYSTEM TABLE", "GLOBAL TEMPORARY", "LOCAL TEMPORARY", "ALIAS", "SYNONYM".
REMARKS String => explanatory comment on the table
TYPE_CAT String => the types catalog (may be null)
TYPE_SCHEM String => the types schema (may be null)
TYPE_NAME String => type name (may be null)
SELF_REFERENCING_COL_NAME String => name of the designated "identifier" column of a typed table (may be null)
REF_GENERATION String => specifies how values in SELF_REFERENCING_COL_NAME are created. Values are "SYSTEM", "USER", "DERIVED". (may be null)

https://javadoc.io/doc/org.postgresql/postgresql/42.0.0.jre7/org/postgresql/jdbc/PgDatabaseMetaData.html#getTables(java.lang.String,%20java.lang.String,%20java.lang.String,%20java.lang.String%5B%5D)